### PR TITLE
53% russian translate

### PR DESCRIPTION
**Pull request by krote5k**
53%

### CODE REVIEW

**Comment by nilmerg** [comment]
Hi Dmitriy,

I've added a bot to the repository which, if enabled, automatically resolves any source catalog conflicts and updates your pull request with our master. If you like to enable this bot for this PR please have a look at the details [here](https://github.com/Icinga/L10n/blob/master/CONTRIBUTING.md#keeping-pull-requests-up-to-date). (Keeping Pull Requests up-to-date)

Though the file `.github/stats.json` will not get resolved by the bot. You don't need to change this file anyway as it's also updated automatically and changes have no effect outside of the master branch. Please undo the change.

Best,
Johannes

**Comment by krote5k** [comment]
Resolve conflicting pull requests0s
    ICINGABOT_TOKEN: 
Run bin/autoresolve
  bin/autoresolve
  shell: /bin/bash -e {0}
  env:
    ICINGABOT_TOKEN: 
Environment variable ICINGABOT_TOKEN not set
##[error]Process completed with exit code 1.

**Comment by nilmerg** [comment]
Weeeell, this action shouldn't run in your fork. Update again and this shouldn't happen.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 1]
```diff
@@ -842,41 +840,37 @@ msgstr ""
...
 msgid "Primary Icinga Master"
 msgstr ""
 
 #: sources.d/icingadb-web/application/forms/RedisConfigForm.php:84
-msgid ""
-"If you are running a high availability zone with two masters, please provide "
-"the Redis connection details of the secondary master."
-msgstr ""
+msgid "If you are running a high availability zone with two masters, please provide the Redis connection details of the secondary master."
+msgstr "Если вы работаете в зоне высокой доступности с двумя мастерами, пожалуйста, предоставьте сведения о соединении Redis вторичного мастера."
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:873]: Same here.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 2]
```diff
@@ -755,11 +753,11 @@ msgstr ""
 #: sources.d/icingaweb2/library/Icinga/Authentication/UserGroup/DbUserGroupBackend.php:123
 #: sources.d/icingaweb2/library/Icinga/Authentication/UserGroup/LdapUserGroupBackend.php:573
 msgid "User Group"
-msgstr ""
+msgstr "Пользовательская группа"
 
 #: sources.d/icingadb-web/application/controllers/UsergroupController.php:37
 msgid "User group not found"
-msgstr ""
+msgstr "Группа пользователей не найдена"
```
[the remark this answers is not in the document]
Comment (Al2Klimov) [reply src/ru_RU/LC_MESSAGES/icinga.po:760]: Note for the future myself: I've intentionally not written "Same here.", yet.

File: src/ru_RU/LC_MESSAGES/icinga.po (hunk 1 above)
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:873]: I read "вы работаете в зоне" like "вы" is referring to me as the user.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 3]
```diff
@@ -896,119 +890,82 @@ msgid "Skip Validation"
...
-msgid "Validation In Progress"
-msgstr ""
+msgid "Check this box to enforce changes without validating that Redis is available."
+msgstr "Установите этот флажок, чтобы применить изменения без проверки доступен ли Redis."
 
 #: sources.d/icingadb-web/application/forms/RedisConfigForm.php:184
 #, php-format
 msgid "Failed to connect to primary Redis: %s"
-msgstr ""
+msgstr "Не удалось подключиться к первичному Radius: %s"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:899]: "Radius"?

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 4]
```diff
@@ -896,119 +890,82 @@ msgid "Skip Validation"
...
-"well."
-msgstr ""
+msgid "Primary connection established but failed to verify Icinga is connected as well."
+msgstr "Первичное соединение установлено, но не удалось проверить, что Icinga также подключена."
 
 #: sources.d/icingadb-web/application/forms/RedisConfigForm.php:212
 #, php-format
 msgid "Failed to connect to secondary Redis: %s"
-msgstr ""
+msgstr "Не удалось подключиться к вторичному Radius: %s"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:908]: Same here.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 5]
```diff
@@ -896,119 +890,82 @@ msgid "Skip Validation"
...
 #: sources.d/icingadb-web/application/views/scripts/form/reorder-command-transports.phtml:68
 #: sources.d/icingaweb2/modules/monitoring/application/views/scripts/form/reorder-command-transports.phtml:80
 #, php-format
 msgid "Move command transport %s downwards"
-msgstr ""
+msgstr "Переместить транспорт команд %s ниже"
 
+#: sources.d/icingadb-web/configuration.php:10
+msgid "Icinga DB"
+msgstr "БД Icinga"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:960]: Same as above.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 6]
```diff
@@ -1053,26 +1011,26 @@ msgstr "Проблемы с узлами"
...
 #: sources.d/icingaweb2/modules/monitoring/configuration.php:344
 msgid "Overdue"
-msgstr ""
+msgstr "Просрочено"
 
 #: sources.d/icingadb-web/configuration.php:79
 #: sources.d/icingaweb2/modules/monitoring/configuration.php:346
 msgid "Late Host Check Results"
-msgstr ""
+msgstr "Поздние результаты проверки узла"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1019]: I'd translate "Host" and "Check" as plural.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 7]
```diff
@@ -1053,26 +1011,26 @@ msgstr "Проблемы с узлами"
...
 
 #: sources.d/icingadb-web/configuration.php:85
 #: sources.d/icingaweb2/modules/monitoring/configuration.php:351
 msgid "Late Service Check Results"
-msgstr ""
+msgstr "Результаты поздней проверки службы"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1024]: ... as well as "Service".

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 8]
```diff
@@ -1053,26 +1011,26 @@ msgstr "Проблемы с узлами"
...
 #: sources.d/icingadb-web/configuration.php:91
 #: sources.d/icingaweb2/modules/monitoring/configuration.php:356
 msgid "Acknowledgements Active For At Least Three Days"
-msgstr ""
+msgstr "Подтверждения действительны не менее трех дней"
 
 #: sources.d/icingadb-web/configuration.php:97
 msgid "Downtimes Active For At Least Three Days"
-msgstr ""
+msgstr "Время простоя активно в течение как минимум трех дней"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1033]: Did you translate "Active For At Least Three Days" different intentionally.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 9]
```diff
@@ -1113,205 +1071,166 @@ msgstr "Обработанные события узлов"
...
 #: sources.d/icingadb-web/library/Icingadb/Common/CommandActions.php:173
 #: sources.d/icingaweb2/modules/monitoring/application/forms/Command/Object/ProcessCheckResultCommandForm.php:31
 msgid "Submit Passive Check Result"
 msgid_plural "Submit Passive Check Results"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "Отправить результат пассивной проверки"
+msgstr[1] "Отправить результата пассивной проверки"
+msgstr[2] "Отправить результатов пассивной проверки"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1117]: Why not "результаты"?

Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1117]: Why didn't you translate "Check" as plural?

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 10]
```diff
@@ -1113,205 +1071,166 @@ msgstr "Обработанные события узлов"
...
 #: sources.d/icingadb-web/library/Icingadb/Common/CommandActions.php:209
 #: sources.d/icingaweb2/modules/monitoring/application/controllers/ServiceController.php:120
 msgid "Schedule Service Downtime"
-msgstr ""
+msgstr "Расписание времени простоя службы"
 
 #: sources.d/icingadb-web/library/Icingadb/Common/CommandActions.php:218
 msgid "Send Custom Notification"
-msgstr ""
+msgstr "Отправить пользовательское уведомление"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1141]: I read the translation as "Send User Notification".

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 11]
```diff
@@ -1113,205 +1071,166 @@ msgstr "Обработанные события узлов"
...
 msgid "unreachable"
-msgstr ""
+msgstr "недоступен"
 
 #: sources.d/icingadb-web/library/Icingadb/Common/HostStates.php:107
 #: sources.d/icingadb-web/library/Icingadb/Common/ServiceStates.php:118
 #: sources.d/icingaweb2-module-director/application/locale/translateMe.php:8
 msgid "pending"
-msgstr ""
+msgstr "рассматриваемый"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1164]: Why not "ожидаемый"?

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 12]
```diff
@@ -1113,205 +1071,166 @@ msgstr "Обработанные события узлов"
...
 msgctxt "<service> on <host>"
 msgid "%s on %s"
-msgstr ""
+msgstr "%s на %s"
 
 #: sources.d/icingadb-web/library/Icingadb/Common/ServiceStates.php:106
 #: sources.d/icingaweb2-module-director/application/locale/translateMe.php:9
 msgid "ok"
-msgstr ""
+msgstr "хор"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1181]: I read the translation as "choir" and would prefer "в порядке".

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 13]
```diff
@@ -1113,205 +1071,166 @@ msgstr "Обработанные события узлов"
...
 msgid "Can't connect to Icinga Redis: %s"
-msgstr ""
+msgstr "Не удается подключиться к Icinga Redis: %s"
 
 #: sources.d/icingadb-web/library/Icingadb/Setup/ApiTransportPage.php:19
 #: sources.d/icingadb-web/library/Icingadb/Setup/ApiTransportStep.php:76
 #: sources.d/icingaweb2/modules/monitoring/application/forms/Config/TransportConfigForm.php:232
 msgid "Icinga 2 API"
-msgstr ""
+msgstr "API Icinga 2"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1218]: Why didn't you even transliterate "API"?

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 14]
```diff
@@ -1113,205 +1071,166 @@ msgstr "Обработанные события узлов"
...
-#: sources.d/icingaweb2/application/forms/Config/UserGroup/UserGroupBackendForm.php:251
-#: sources.d/icingaweb2/modules/monitoring/application/forms/Config/TransportConfigForm.php:369
-#: sources.d/icingaweb2/modules/monitoring/application/forms/Setup/IdoResourcePage.php:143
-#: sources.d/icingaweb2/modules/setup/application/forms/AuthBackendPage.php:214
-#: sources.d/icingaweb2/modules/setup/application/forms/DbResourcePage.php:117
-#: sources.d/icingaweb2/modules/setup/application/forms/LdapResourcePage.php:119
-#, php-format
-msgid "Failed to successfully validate the configuration: %s"
-msgstr ""
+msgstr "Пожалуйста, заполните информацию о подключении к API Icinga 2."
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1222]: Same here.

File: src/ru_RU/LC_MESSAGES/icinga.po [hunk 15]
```diff
@@ -1113,205 +1071,166 @@ msgstr "Обработанные события узлов"
...
 
 #: sources.d/icingadb-web/library/Icingadb/Setup/ApiTransportPage.php:107
 #: sources.d/icingadb-web/library/Icingadb/Setup/DbResourcePage.php:141
 #: sources.d/icingaweb2/modules/monitoring/application/forms/Setup/IdoResourcePage.php:184
 #: sources.d/icingaweb2/modules/setup/application/forms/DbResourcePage.php:179
 msgid "Check this to not to validate the configuration"
-msgstr ""
+msgstr "Проверьте это, чтобы не проверять конфигурацию"
```
Comment (Al2Klimov) [inline src/ru_RU/LC_MESSAGES/icinga.po:1229]: No, not "Check" like _verify_, but like check a checkbox.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Hello @krote5k and thank you for contributing!

Please don't take my complaints at face value. Yes, I'm 🇷🇺, but... w/o auto-completion I'd be literally lost in 🇷🇺 Twitter. So let's discuss my complaints before just correcting everything.

One general complaint of mine you've translated host, service, pager, ... but you didn't even transliterate Icinga or Redis? AFAIK we 🇷🇺 transliterate lots of stuff (e.g. "fake" in context of fake news) – just because we 🇷🇺 don't have these letters. How you'd do this?

Oh and I've not always written "Same here", so please lookup similar errors by yourself.

Best,
AK

Note for the future myself: I've stopped at "The Redis module for PHP in version 4.3 and above requires PHP 7+" *Sigh.* So much to review and so less time...

**Comment by nilmerg** [comment]
Hi Dmitriy,

we'd like to finally include your work in the upcoming releases, albeit being incomplete. Do you still have uncommitted progress? If so, please push that so it will not be forgotten.

Either way though, I'll merge this soon enough. You can then continue in a separate (new) PR if you'd like. :smiley: 

Best,
Johannes

**Comment by krote5k** [comment]
I agree. I don't know if it will be possible to continue

**Comment by nilmerg** [comment]
Superseded by #44

**Pull request closed without merging**